### PR TITLE
[NUXE-206] Unify use of error/error_message props across input kits

### PR DESCRIPTION
**Pull request by maddiemc01**
#### Screens
checkbox:
Screen Shot 2020-10-27 at 12 10 11 PM
text input:
Screen Shot 2020-10-26 at 4 25 20 PM
date picker:
Screen Shot 2020-10-26 at 4 25 54 PM
text area:
Screen Shot 2020-10-27 at 9 15 22 AM
radio:
Screen Shot 2020-10-27 at 10 41 52 AM
select:
Screen Shot 2020-10-27 at 12 42 13 PM

#### Breaking Changes

YES. This is breaking because the error prop was previously a string, but in the version has changed to a boolean for consistency. The breaking kits are: `textarea`, `text_input`, `checkbox`, `date_picker`, and `select`. Radio kit is an addition.

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUXE-206

#### How to test this

Check that these kits display an error intended and that the form kit generates propper real-time errors:
checkbox
text_input
date_picker
radio
textarea
select

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **URGENCY** Please select a release milestone
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: app/pb_kits/playbook/pb_radio/_radio.jsx [hunk 1]
```diff
@@ -9,17 +9,18 @@ import { globalProps } from '../utilities/globalProps.js'
...
+  checked?: boolean,
   children?: Node,
-  className?: String,
+  className?: string,
   dark?: boolean,
   data?: object,
-  error?: Boolean,
-  id?: String,
-  label: String,
-  name: String,
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_radio/_radio.jsx:20]: I love all the fixes to the flow types!

File: milano.production.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ deploy:
...
-  metrics: "https://grafana-hq.powerapp.cloud/d/hYTBQhQiz/pod-resource-usage?orgId=1&refresh=10s&var-namespace=playbook-$ENVIRONMENT&var-filter=.%2B&var-pod=All"
+  logs: "https://logging.prod.hq.powerapp.cloud/app/kibana#/discover?_g=(refreshInterval:(pause:!f,value:5000),time:(from:now-15m,mode:quick,to:now))&_a=(columns:!(log),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:ae758a80-1475-11eb-a138-277caf074f3e,key:kubernetes.namespace_name,negate:!f,params:(query:playbook-production,type:phrase),type:phrase,value:playbook-production),query:(match:(kubernetes.namespace_name:(query:playbook-production,type:phrase))))),index:ae758a80-1475-11eb-a138-277caf074f3e,interval:auto,query:(language:lucene,query:''),sort:!('@timestamp',desc))"
+  metrics: "https://metrics.prod.hq.powerapp.cloud/d/hYTBQhQiz/pod-resource-usage?orgId=1&refresh=10s&var-datasource=prometheus-app-prod-hq&var-namespace=playbook-production&var-filter=.%2B&var-pod=All"
```
Comment (garettarrowood) [inline milano.production.yml:11]: Is this change intentional?

Comment (maddiemc01) [reply milano.production.yml]: i didnt make it on purpose so... im gonna guess no but looks like it got fixed some how

**Review by jasoncypret — APPROVED** [review]

**Comment by jasoncypret** [comment]
Release notes:

- [ ] Needs rebased.
- [ ] Select is not displaying correctly.
- [ ] Form kit submit doesn't work properly.
- [ ] Need a Nitro list for each kit to correct with the release

**Comment by jasoncypret** [comment]
After looking at this one again and how much testing work will be involved I'd like to break this update per kit and one release at a time. We will keep this PR open as a reference. The checkbox can go immediately as it is not a breaking change.

**Comment by kre8sions** [comment]
Closing PR but keeping the branch alive @jasoncypret
